python/lsst/the/monster/refcats.py

```diff
@@ -479,3 +479,18 @@ def get_sn_range(self, band):
             return (20.0, np.inf)
         else:
             return (0.0, np.inf)
+
+
+class SynthLSSTInfo(RefcatInfo):
+    NAME = "SynthLSST"
+    bands = ["u", "g", "r", "i", "z", "y"]
+
```

Comment:
I could imagine wanting a version number e.g 1.9 as an attribute for the` SynthLSSTInfo`, but not sure the best way to do that or if it would be worth it.

Reply:
I think that it's up to us as we do a release of new versions of the monster.  And really it doesn't matter that much; we'll start with some reasonable synth and then update when we have real data.